src/cowboy_swagger.erl

```diff
@@ -6,36 +6,36 @@
 
 %% Utilities
 -export([enc_json/1, dec_json/1]).
--export([swagger_paths/1]).
+-export([swagger_paths/1, validate_metadata/1]).
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %% Types.
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
--opaque swagger_parameters() ::
-  #{ name        => binary()
-   , in          => binary()
-   , description => binary()
+-opaque parameter_obj() ::
+  #{ name        => binary() | string()
```

Comment:
Is this necesary? Can't we just agree to use one or the other for all cases?

And if not, why not just use ``iodata()``?